airbyte-integrations/connectors/source-posthog/sample_files/configured_catalog.json

```diff
@@ -0,0 +1,631 @@
+{
```

Comment:
actually there is no need to have both, we need only one the used in SAT tests, I understand this one probably was autogenerated, but lets remove it